src/en/juju-ha.md

```diff
@@ -51,16 +51,10 @@ state servers nor remove the failed ones. However, as long as more than half of
 the original number of state servers remain available you can manually recover.
 The process is detailed below.
 
-1. Run `juju ensure-availability`.
-1. Verify that the output of `juju status` shows a value of `has-vote` for the
```

Comment:
Nope. I can't remove old controllers at all. I can add to my original pool of 3, up to 5 or 7 total. Perhaps controllers have to be in a 'failed' state before they can be removed. I can not produce this state.